mico-core/src/main/java/io/github/ust/mico/core/broker/MicoServiceDeploymentInfoBroker.java

```diff
@@ -116,7 +116,8 @@ public MicoServiceDeploymentInfo getExistingServiceDeploymentInfo(MicoApplicatio
         if (serviceDeploymentInfos.size() > 1) {
             String errorMessage = "There are " + serviceDeploymentInfos.size() + " service deployment information stored for service '" +
                 micoService.getShortName() + "' '" + micoService.getVersion() + "' used by application '" + micoApplication.getShortName() + "' '" +
-                micoApplication.getVersion() + "'. However, there must be only one.";
+                micoApplication.getVersion() + "': '" + serviceDeploymentInfos.stream().map(MicoServiceDeploymentInfo::getInstanceId).collect(Collectors.toList()) +
```

Comment:
maybe use local variable to give a name to this long stream processing for easier understanding